.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
 test/coverage
+.idea
+*.iml
```

Comment:
@lukas-schardt I'm not a big fan of these editor specific files in here. I would encourage everyone to put these in a global .gitignore on your computer.